app/controllers/waste_carriers_engine/renewal_start_forms_controller.rb

```diff
@@ -14,16 +14,13 @@ def create
 
     private
 
+    # rubocop:disable Naming/MemoizedInstanceVariableName
```

Comment:
It does, but renaming it would possibly be confusing as the current name is very explicit. Note that this `rubocop:disable` statement was previously in place and I have only reinstated it. Note also that it is present in several places in the code, in one instance with this comment:

\# We're not really memoizing this instance variable here, so we don't think
\# this cop is valid in this context

... which seems fair enough.